### PR TITLE
Update README to make dev setup easier.

### LINKED ISSUE #388: Update README Badges

### PR DESCRIPTION
**Pull request by tpendragon**
Closes #388

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,39 +1,50 @@
...
-```
-rake set_admin_role
-```
-You can also specify an email address to make a particular user an admin:
-```
-EMAIL=user@example.org rake set_admin_role
+```sh
+git clone https://github.com/pulibrary/cicognara-rails.git
+cd cicognara-rails
+bundle install
```
Comment (hackmastera) [inline README.md]: then yarn install here?

Comment (tpendragon) [reply README.md]: Project's too old, it doesn't use yarn/npm/webpacker yet.

Comment (hackmastera) [reply README.md]: Ah, okay, then the line below that mentions `yarn install` needs to be changed.

Comment (tpendragon) [reply README.md]: Yup, fixed. Added info about getty import while I was it.

**Pull request merged**